lib/middleware/production.js

```diff
@@ -0,0 +1,22 @@
+// This Express middleware function sets some
+// HTTP headers which should only be set in deployed
+// environments
+
+function production(req, res, next) {
+  // Set Strict-Transport-Security header to
+  // ensure that browsers only use HTTPS
+  res.setHeader('Strict-Transport-Security', 'max-age=31536000; includeSubDomains; preload');
```

Comment:
Did we decide these didn't work unless `herokuapp.com` added headers too?

```suggestion
  res.setHeader('Strict-Transport-Security', 'max-age=31536000');
```

Reply:
We did, but they might be worth keeping in case you deploy elsewhere, or use a custom top-level domain on Heroku or elsewhere. (Unlikely but you never know).

I don’t think they’ll do any harm when used on a subdomain, they’ll just be ignored?